### PR TITLE
Update TFLu (micro) to use AmbiqSuite SDK Release 2.2.0 for Apollo3

### PR DESCRIPTION
**Pull request by oclyke**
We can update to the latest SDK release from Ambiq to stay current. This will make it easier to build/maintain additional functionality in the future if needed.

### CODE REVIEW

**Review by petewarden — APPROVED** [review]

**Comment by oclyke** [comment]
@gbaned Should I be concerned about the Linux GPU, Windows Bazel, and Windows Bazel GPU checks failing? If you can advise me on next steps to land this PR I would appreciate it much. 

Thanks!

**Comment by gbaned** [comment]
> @gbaned Should I be concerned about the Linux GPU, Windows Bazel, and Windows Bazel GPU checks failing? If you can advise me on next steps to land this PR I would appreciate it much.
> 
> Thanks!

@oclyke Sorry for the delay, it is waiting for an internal approval. We will let you know if anything requires from you for this PR. Thank you!

**Pull request merged**
